apps/drizzle/src/select.ts

```diff
@@ -44,19 +44,19 @@ const fnForOp: (op: string) => BinaryOperator = (op) => {
 }
 
 const csEq: BinaryOperator = (left: SQLWrapper, right: unknown): SQL => {
-  return sql`cs_unique_v1(${left}) = cs_unique_v1(${bindIfParam(right, left)})`
+  return sql`cs_unique_v2(${left}) = cs_unique_v2(${bindIfParam(right, left)})`
```

Comment:
Functions are now under the `eql_v2` schema: `eql_v2.unique()`

The wrapping into functions may not be needed. 
If the types are cast to `eql_v2_encrypted` the operators should work:

```
'{  }'::jsonb::eql_v2_encrypted
```

Reply:
I'll do a follow up PR for the Drizzle example